src/content/resources/concepts/layer/assessment/assessment-results/_index.md

```diff
@@ -13,7 +13,7 @@ aliases:
 
 | Assessment Results Schema | Assessment Results Converters | Reference
 |:--- |:--- |:--- |
-| [JSON Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_assessment-results_schema.json) | [XML to JSON Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_assessment-results_xml-to-json-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/tree/main/json#converting-oscal-xml-content-to-json)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-results/json-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-results/json-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-results/json-index/) |
+| [JSON Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_assessment-results_schema.json) | [XML to JSON Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_assessment-results_xml-to-json-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/blob/main/build/README.md#converters)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-results/json-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-results/json-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-results/json-index/) |
 | [XML Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_assessment-results_schema.xsd) | [JSON to XML Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_assessment-results_json-to-xml-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/tree/main/xml#converting-oscal-json-content-to-xml)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-results/xml-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-results/xml-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-results/xml-index/) |
```

Comment:
Link https://github.com/usnistgov/OSCAL/tree/main/xml#converting-oscal-json-content-to-xml is broken. Needs to also be replaced.

Reply:
I figured out how I missed this, I rebased the half of links that were incorrect (I completed the XML to JSON link replacement, but not the JSON to XML ones you cited). I rebased the original commit in the series and this was addressed.